ChangeLog.md

```diff
@@ -3,6 +3,316 @@
+- Balance Hyperspatial Dam
+
+
+#### Bugs
+
+- Prevented dragging sitrep entries
+- Fixed depopulation of planets to work when planet has a negative population
+- Fixed rendering of object window items selected with right click
+- Fixed multiplayer galaxy setup droplists problematic scrollbars
+- Fixed protection focus effect on max troops
+- Fixed AI building neutronium extractors
+- Fixed AI problem with ship design with id 0
+- Fixed experimentor spawn location being blocked by natives
+- Fixed a segfault when resigning a game with no empire
+- Fixed Fortress special mines effects not properly working together
+with System Defense Mines techs
+- Prevented unowned planet mines from affecting unowned guard ships
+- Fixed unowned fortress mines damaging unowned monsters and made
+mines able to destroy monsters, not just damage them
+- Fixed ion storm stealth effect to not affect fields (and thereby itself)
+- Fixed issue with stealth effect of Transpatial Drive
+- Reduced credits scroll flicker
+- Fixed quirks with list scrollbars and row positioning
+- Fixed crash when manipulating fleets in Fleets window
+- Fixed issue where saves would record an extra nonexistant player,
+causing problems when loading the save
+- Fixed issue with dragging parts over another part in a design
+- Prevent a bug with high CPU load and low FPS. A font is now cached
+even if a requested glyph is not part of the font, but a 'replacement
+character' is provided
+- Fixed crash when setting hotkeys
+- Fixed a bug where calling the suitability report from the production
+window would toggle the production window pedia panel visibility. Now
+shows suitability report in production window pedia.
+- Fixed alphabetical sorting of object list columns when accented
+letters or other non-latin characters are present
+- Fixed a bug where pedia closed instead of showing an entry
+- Fixed deleting production items by pressing delete
+- Fixed crashes when clicking links in pedia
+- Fixed calculation of FPS
+- Fixed resource panel not showing expansion
+- Fixed a bug where the AI would sometimes think it can't build parts
+at a planet despite it actually being able to
+- Fixed fleet destination text being "holding at ..." when moving
+along a path that returns to the start location.
+- Fixed xenophobic self-sustaining pop malus and xenophobic frenzy
+happiness malus
+
+
+#### Technical / Internal
+
+- Made it possible to read binary or compressed XML saves regardless
+of current save output format setting
+- Made communication between clients / servers on different operating
+systems more flexible / robust
+- Reworked XML save format to have an uncompressed somewhat
+readable-as-text header with basic info about the saved game
+- Added --version parameter to freeorion{,d} executables which outputs
+the version string to the console
+- (*nix) Moved user config and data files to conform with
+XDG Base Directory Specification
+- Reorganized content definitions into multiple files
```

Comment:
Ok, very concise, but not sure if that's really, hm, sufficiently "prominent"... if anyone has better ideas, speak up 😉